modules/nf-core/fastqscreen/fastqscreen/meta.yml

```
@@ -0,0 +1,49 @@
+name: fastqscreen_fastqscreen
+description: Align reads to multiple reference genomes using fastq-screen
+keywords:
+  - align
+  - map
+  - fasta
+  - fastq
+  - genome
+  - reference
+tools:
+  - "fastqscreen":
+      description: "FastQ Screen allows you to screen a library of sequences in FastQ format against a set of sequence databases so you can see if the composition of the library matches with what you expect."
+      homepage: "https://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/"
+      documentation: "https://stevenwingett.github.io/FastQ-Screen/"
+      tool_dev_url: "https://github.com/StevenWingett/FastQ-Screen/archive/refs/tags/v0.15.3.zip"
+      doi: "10.5281/zenodo.5838377"
+      licence: ["GPL-3.0-or-later"]
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - reads:
+      type: file
+      description: |
+        List of input FastQ files of size 1 and 2 for single-end and paired-end data,
+        respectively.
+  - meta2:
+      type: map
+      description: |
+        Groovy Map containing reference information
```

```suggestion
        Groovy Map containing information about the reference fasta. Must include id, which becomes the name used in the tool output.
```
This may be untrue as I haven't tested the module, but looks like what will happen and my memory of using the tool in the past.